test/e2e/run_cgroup_parent_test.go

```diff
@@ -48,21 +48,22 @@ var _ = Describe("Podman run with --cgroup-parent", func() {
 		run := podmanTest.Podman([]string{"run", "--cgroupns=host", "--cgroup-parent", cgroup, fedoraMinimal, "cat", "/proc/self/cgroup"})
 		run.WaitWithDefaultTimeout()
 		Expect(run).Should(Exit(0))
-		ok, _ := run.GrepString(cgroup)
-		Expect(ok).To(BeTrue())
+		Expect(run.OutputToString()).To(ContainSubstring(cgroup))
```

Comment:
I really can't begin to express how anti-helpful it is to see "zero is not one" in ginkgo error logs.